contextConsumption/query_entities_test.js

```diff
@@ -1,14 +1,14 @@
-'use strict';
 
-var testedResource = require('../common.js').testedResource;
-var http = require('../http.js');
 
-var entitiesResource = testedResource + '/' + 'entities' + '/';
-var assertRetrievedQuery = require('../common.js').assertRetrievedQuery;
-var serializeParams = require('../common.js').serializeParams;
+const testedResource = require('../common.js').testedResource;
+const http = require('../http.js');
+
+const entitiesResource = testedResource + '/entities/';
+const assertRetrievedQuery = require('../common.js').assertRetrievedQuery;
+const serializeParams = require('../common.js').serializeParams;
 
 describe('Query Entity. JSON. Default @context', () => {
-  let entity = {
+  const entity = {
```

Comment:
is it really needed to change from let to const? 

From my point of view that is not a const just a variable in your code

Reply:
NTC - its a variable that never changes - therefore its a `const`. 
Don't blame me - Its not me who amends the code - it is *prettier* changing it on the fly